.circleci/config.yml

```diff
@@ -101,7 +101,7 @@ jobs:
       - store_artifacts:
           path: test-reports
       - store_artifacts:
-          path: cover.tar.gz
+          path: cover # TODO .tar.gz
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker